ci/build_and_test.sh

```diff
@@ -18,10 +18,10 @@ echo "Running isort"
 isort -c .
 echo "Running bandit"
 bandit -q -ll --recursive merlin_models
-echo "Running pylint"
-pylint merlin_models tests
-echo "Running flake8-nb"
-flake8-nb examples
+# echo "Running pylint"
+# pylint tests
+# echo "Running flake8-nb"
+# flake8-nb examples
 
 # test out our codebase
-py.test --cov-config tests/unit/.coveragerc --cov-report term-missing --cov-report xml --cov-fail-under 70 --cov=. tests/tensorflow
\ No newline at end of file
+py.test --cov-config tests/.coveragerc --cov-report term-missing --cov-report xml --cov-fail-under 70 --cov=. tests
```

Comment:
How about we just run the TF tests for now?

```suggestion
py.test --cov-config tests/.coveragerc --cov-report term-missing --cov-report xml --cov-fail-under 70 --cov=. tests/tensorflow
```